tests/manual-test-cases/Group19-ROBO/19-1-ROBO-SKU.md

```diff
@@ -1,24 +1,24 @@
-Test 19-1 - ROBO SKU
+Test 5-19 - ROBO SKU
 =======
 
 # Purpose:
-To verify that VIC works properly when a VCH is installed in a remote office branch office (ROBO) version of vSphere.
+To verify that VIC works properly when a VCH is installed in a remote office branch office (ROBO) version of vSphere
 
 # References:
-1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+1 - [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
 
 # Environment:
 This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation
 
 # Test Steps:
-1. Deploy a new vCenter with stand alone hosts
-2. Add the Enterprise license to the vCenter appliance
+1. Deploy a simple vSAN cluster
+2. Add the ROBO SKU license to the vCenter appliance
 3. Assign the ROBO SKU license to each of the hosts within the vCenter
-4. Install a VCH onto a particular multi-host cluster in the vCenter
```

Comment:
We should retain steps 4 and 5 as they were previously, unless we're supposed to change them in the PR.